packages/config/src/layer3s/deri.ts

```diff
@@ -1,14 +1,13 @@
 import { ProjectId } from '@l2beat/shared-pure'
 
-import { CONTRACTS, TECHNOLOGY, UNDER_REVIEW_RISK_VIEW } from '../common'
-import { Layer2 } from '../types'
+import { CONTRACTS } from '../layer2s/common'
+import { Layer3 } from './types'
 
-export const deri: Layer2 = {
+export const deri: Layer3 = {
+  type: 'layer3',
   isUpcoming: false,
   isUnderReview: true,
   isArchived: false,
```

Comment:
i dont know if all of them are needed, also it could be:
`status: 'upcoming' | 'under review' | 'archived'`
as probably you never want both of these states at the same time